terratorch/models/backbones/clay_v1/embedder.py

```diff
@@ -0,0 +1,173 @@
+import re
+import warnings
+
+import numpy as np
+import torch
+from torch import nn, Tensor
+import torch
+from timm.models import FeatureInfo
+from timm.models._builder import build_model_with_cfg
+from timm.models._registry import generate_default_cfgs, register_model
+
+from terratorch.models.backbones.clay_v1.modules import EmbeddingEncoder, Datacuber
+
+warnings.filterwarnings("ignore", category=UserWarning)
+
+
+default_cfgs = generate_default_cfgs(
+    {
+        "clay_v1_base": {
+            "hf_hub_id": "made-with-clay/Clay",
+            "hf_hub_filename": "clay-v1-base.ckpt"
+        }
+    }
+)
+
+
+class Embedder(nn.Module):
+    default_out_indices = (0,)  # Single out_indices for simplicity
+
+    def __init__(self,
+                 img_size=256,
+                 num_frames=1,
+                 ckpt_path=None,
+                 device="cuda",
+                 bands=["blue", "green", "red", "nir", "swir16", "swir22"],
+                 **kwargs):
+        super().__init__()
+        self.feature_info = []
+        self.img_size = img_size
+        self.num_frames = num_frames
+        self.bands = bands
+
+        if kwargs.get("datacuber", True) is not None:
+            self.datacuber = Datacuber(bands=bands)
+        else:
+            self.datacuber = None
+            
+        # TODO: add support for various clay versions
+        self.clay_encoder = (
+            EmbeddingEncoder(  # Default parameters for the Clay base model
+                img_size=img_size,
+                patch_size=8,
+                dim=768,
+                depth=12,
+                heads=12,
+                dim_head=64,
+                mlp_ratio=4.0,
+            ).to(device)
```

Comment:
Same as above, such statements should be removed

Reply:
Makes sense. Those have been removed.